### PR TITLE
SAIC-339 Readable functional tests error messages

### PR DESCRIPTION
**Pull request by roman-verchikov**
Removes lower-level libraries debug logging from nosetests reports

### CODE REVIEW

File: devlab/tests/functional_test.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+                       'requests.packages.urllib3.connectionpool']
+
+    for l in suppressed_logs:
+        logging.getLogger(l).setLevel(logging.WARNING)
+
+
+class FunctionalTest(unittest.TestCase):
+    def __init__(self, *args, **kwargs):
+        super(FunctionalTest, self).__init__(*args, **kwargs)
+        suppress_dependency_logging()
```
Comment (Settis) [inline devlab/tests/functional_test.py]: Add blank line at the end of file according PEP8.

**Comment by Settis** [comment]
Looks good. +1 from my side.

**Comment by realsystem** [comment]
+1

File: devlab/tests/functional_test.py (hunk 1 above)
Comment (roman-verchikov) [inline devlab/tests/functional_test.py]: Hmmm... pep8 didn't complain:
```
$ pep8 devlab/tests/functional_test.py
$
```

**Comment by roman-verchikov** [comment]
can be merged

**Pull request merged**